src/DashboardHub/Bundle/AppBundle/Service/TravisService.php

```diff
@@ -47,9 +48,34 @@ public function getBuilds($reponame, $max = 5)
                 ,
                 true
             );
-            $cache->set($builds, 600);
+            $cache->set($builds, 60);
```

Comment:
In branch or as a new pr?

On Tue, Mar 17, 2015 at 4:38 PM, Eddie Abou-Jaoude <notifications@github.com
> wrote:

> In src/DashboardHub/Bundle/AppBundle/Service/TravisService.php
> <https://github.com/DashboardHub/PipelineDashboard/pull/98#discussion_r26589987>
> :
>
> > @@ -47,9 +48,34 @@ public function getBuilds($reponame, $max = 5)
> >                  ,
> >                  true
> >              );
> > -            $cache->set($builds, 600);
> > +            $cache->set($builds, 60);
>
> Yep, I will sort it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DashboardHub/PipelineDashboard/pull/98/files#r26589987>
> .
>